src/api/FetchApi.js

```diff
@@ -0,0 +1,26 @@
+const fetch = require("node-fetch");
+
+module.exports = class FetchApi {
+    constructor(baseUrl) {
+        this.baseUrl = baseUrl ? baseUrl : "https://api.github.com/";
```

Comment:
I'm not sure if it's good idea define this URL on this file. Maybe it's better to do it in the `GitHubApi.js` file

Reply:
yes! you are right!! I'll change it